functions/get-team.ts

```diff
@@ -33,7 +33,7 @@ exports.handler = async (event) => {
       };
     } else {
       return {
-        statusCode: 401,
+        statusCode: 200,
```

Comment:
I actually updated it to a status 204.

I think it would makes sense as it would mean that the call was successfull but there is no body to the request. What do you think ?